js/packages/openinference-vercel/src/OpenInferenceVercelSpanProcessor.ts

```diff
@@ -0,0 +1,32 @@
+import { ReadableSpan, SpanProcessor } from "@opentelemetry/sdk-trace-base";
+import {
+  getOISpanKindFromAttributes,
+  getOpenInferenceAttributes,
+} from "./utils";
+import { SemanticConventions } from "@arizeai/openinference-semantic-conventions";
+import { ReadWriteSpan } from "./types";
+
+export class OpenInferenceVercelSpanProcessor implements SpanProcessor {
```

Comment:
```suggestion
export class VercelSpanProcessor implements SpanProcessor {
```

I think shorter the better

Reply:
good call

Reply:
Or is it the reverse... OpenInferenceSpanProcessor. I guess whichever is less confusing.